src/io/serialization/v2.0.0/serialize.ts

```diff
@@ -1,35 +1,24 @@
-import { trim, isEqual } from 'lodash'
+/* eslint-disable camelcase */
+import { trim } from 'lodash'
 
 import Ajv from 'ajv'
 import ajvLocalizers from 'ajv-i18n'
 
-import validateShareable, { errors } from '../../../.generated/latest/validateShareable'
+import validateShareable, { errors } from '../../../.generated/2.0.0/validateShareable'
+import { Convert } from '../../../.generated/2.0.0/types'
 
 import type { ScenarioParameters, Shareable } from '../../../algorithms/types/Param.types'
-import { Convert } from '../../../algorithms/types/Param.types'
-import { toExternal, toInternal } from '../../../algorithms/types/convert'
 
 import { DeserializationErrorConversionFailed, DeserializationErrorValidationFailed } from '../errors'
+import v2_1_0 from '../v2.1.0/serialize'
 
-const schemaVer = '2.0.0'
+const schemaVer = '2.0.0' as const
+const schemaVerNext = '2.1.0' as const
 
-function serialize(scenarioParameters: ScenarioParameters): string {
-  const shareable: Shareable = {
-    ...scenarioParameters,
-    schemaVer,
-    scenarioData: {
-      ...scenarioParameters.scenarioData,
-      data: toExternal(scenarioParameters.scenarioData.data),
-    },
-  }
-
-  const serialized = Convert.shareableToJson(shareable)
-
-  if (process.env.NODE_ENV !== 'production' && !validateShareable(JSON.parse(serialized))) {
-    throw errors
-  }
-
-  return serialized
+export function serialize(_0: ScenarioParameters): string {
+  throw new Error(`: Developer error: This function should never be called.
+  Serialization must always use the latest version of the serializer, but serializer of version "${schemaVer}" is called instead.
+  Change the callee to use the latest version.`)
 }
```

Comment:
Now that we have serializer of v2.1.0, the old serializer (v2.0.0) should never be called, as we always want to output files and URLs in the new format.